### PR TITLE
fail fast on STS accounts attempting to rotate credentials

### PR DESCRIPTION
**Pull request by iamkirkbater**
This fails fast in order to prevent chasing red-herrings when attempting to rotate IAM user credentials on STS-based accounts.

Resolves [OSD-15115](https://issues.redhat.com//browse/OSD-15115)

### CODE REVIEW

**Comment by iamkirkbater** [comment]
/hold

I haven't had a chance to manually test this

**Review by mjlshen — APPROVED** [review]
/lgtm

Pending testing ;) but agreed that this should not be run on STS clusters

**Review by fahlmant — APPROVED** [review]
/approve

**Comment by iamkirkbater** [comment]
/hold

While I was in there testing this I noticed the help text needed updating to make it more clear how to use these commands.  Will need another lgtm.  Thanks!

**Pull request merged**
